SDL/BinSkimSetup.ps1

```diff
@@ -0,0 +1,38 @@
+﻿[CmdletBinding()]
+Param(
+    [Parameter(Mandatory=$True)]
+    [string]$buildDirectory,
+    
+    [Parameter(Mandatory=$True)]
+    [string]$binSkimDirectory
+)
+
+#$buildDirectory = "C:\Repos\SDK\bin\Release\Src\"
```

Comment:
Suggestion: 
You can add default values to the parameters to make testing easier. Perhaps something like:

    [string]$buildDirectory=(Join-Path -Path (Split-Path -parent $PSCommandPath) -ChildPath "bin\Release\Src")

...might make testing locally a bit more handy.

Reply:
THAT would make things easier!
But i need to go up three parents. Is there an easier way than this:

```
[string]$buildDirectory = (Join-Path -Path (Split-Path -parent (Split-Path -parent (Split-Path -parent $PSCommandPath))) -ChildPath "bin\Debug\Src") 
```